src/web/components/App.tsx

```diff
@@ -294,7 +294,8 @@ export const App = ({ CAPI, NAV }: Props) => {
 				loadScript('https://www.google-analytics.com/analytics.js');
 				loadScript(window.guardian.gaPath);
 			} else {
-				(window as any).ga = null;
+				// @ts-ignore
+				window.ga = null;
```

Comment:
if `consentGiven` is false we set `window.ga = null`, even though this is an illigal operation in TS, so using `// @ts-ignore` makes sense here